yazi-shared/src/fs/file.rs

```diff
@@ -62,6 +62,23 @@ impl File {
 		Self { url, cha: Cha::from(meta).with_kind(ck), link_to, icon: Default::default() }
 	}
 
+	pub async fn from_no_meta(url: Url, file_type: Option<FileType>) -> Self {
+		// Failed to get metadata for this file, so marking it broken
+		let mut ck = ChaKind::ORPHAN;
```

Comment:
We can't use `ORPHAN` to mark it as a broken file, because `ORPHAN` indicates a normal link file whose file linked to doesn't exist, not a file that itself is inaccessible for metadata (but exists and is a non-link file).